awx_collection/plugins/module_utils/controller_api.py

```
@@ -782,11 +799,53 @@ def _authenticate_with_basic_auth(self):
+                except Exception as e:
+                    resp = 'unknown {0}'.format(e)
+                self.fail_json(msg='Failed to get token: {0}'.format(he), response=resp)
+            except (Exception) as e:
+                self.fail_json(msg='Failed to get token: {0}'.format(e))
+
+            token_response = None
+            try:
+                token_response = response.read()
+                response_json = loads(token_response)
+                self.oauth_token = response_json['token']
+            except (Exception) as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)